### DISCUSSION TITLE
DMS look for mail.mydomain.com-full.pem even when I set SSL_KEY_CERT (and KEY)

### DISCUSSION THREAD

**Discussion by Lyxass** [docker-mailserver/docker-mailserver#3545] [Q&A] [answered]
Hello, 
I'm trying to configure a mailserver using DMS however, I'm facing an issue with the TLS configuration. I want to use my custom certificate (which is generated by my DNS provider) so I set SSL_TYPE to custom. I also set the variables SSL_CERT_PATH and SSL_KEY_PATH. But for an unknown reason, It seems to looking for a certificate named mail.mydomain.com-full.pem and I have no clue to understand the problem

Did someone have an idea of what's happened?

Here is my docker-compose configuration :

`version: '3.8'

services:
  mailserver:
    image: ghcr.io/docker-mailserver/docker-mailserver:latest
    container_name: mailserver
    # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
    hostname: mail.mydomain.com
    environment: 
      - DMS_DEBUG=0
      - LOG_LEVEL=info
      - SUPERVISOR_LOGLEVEL=info
      - ONE_DIR=1
      - POSTMASTER_ADDRESS=me@mydomain.com
      - ENABLE_UPDATE_CHECK=1
      - UPDATE_CHECK_INTERVAL=1d
      - PERMIT_DOCKER=none
      - TZ=Europe/Paris
      - ENABLE_SRS=0
      - ENABLE_OPENDKIM=1
      - ENABLE_OPENDMARC=1
      - ENABLE_POLICYD_SPF=1
      - ENABLE_CLAMAV=0
      - ENABLE_RSPAMD=0
      - RSPAMD_LEARN=0
      - RSPAMD_CHECK_AUTHENTICATED=0
      - RSPAMD_GREYLISTING=0
      - RSPAMD_HFILTER=1
      - RSPAMD_HFILTER_HOSTNAME_UNKNOWN_SCORE=6
      - ENABLE_AMAVIS=1
      - AMAVIS_LOGLEVEL=0
      - ENABLE_DNSBL=0
      - ENABLE_FAIL2BAN=0
      - FAIL2BAN_BLOCKTYPE=drop
      - POSTSCREEN_ACTION=enforce
      - SSL_TYPE=custom
      - SSL_CERT_PATH=/tmp/dms/custom-certs/mydomain.com_ssl_certificate.cer
      - SSL_KEY_PATH=/tmp/dms/custom-certs/mydomain.com_private_key.key
      - ENABLE_QUOTAS=1
      - LOGROTATE_INTERVAL=weekly
      - POSTFIX_REJECT_UNKNOWN_CLIENT_HOSTNAME=0
      - POSTFIX_INET_PROTOCOLS=all
      - DOVECOT_INET_PROTOCOLS=all
      - ENABLE_SPAMASSASSIN=0
      - SPAMASSASSIN_SPAM_TO_INBOX=1
      - ENABLE_SPAMASSASSIN_KAM=0
      - MOVE_SPAM_TO_JUNK=1
      - MARK_SPAM_AS_READ=0
      - SA_TAG=2.0
      - SA_TAG2=6.31
      - SA_KILL=10.0
      - SA_SPAM_SUBJECT=***SPAM*****
      - ENABLE_FETCHMAIL=0
      - FETCHMAIL_POLL=300
      - ENABLE_GETMAIL=0
      - GETMAIL_POLL=5
      - DOVECOT_MAILBOX_FORMAT=maildir
      - ENABLE_POSTGREY=0
      - POSTGREY_DELAY=300
      - POSTGREY_MAX_AGE=35
      - POSTGREY_TEXT="Delayed by Postgrey"
      - POSTGREY_AUTO_WHITELIST_CLIENTS=5
      - ENABLE_SASLAUTHD=0
      - SRS_SENDER_CLASSES=envelope_sender
      - RELAY_PORT=25
    # More information about the mail-server ports:
    # https://docker-mailserver.github.io/docker-mailserver/latest/config/security/understanding-the-ports/
    # To avoid conflicts with yaml base-60 float, DO NOT remove the quotation marks.
    ports:
      - "25:25"    # SMTP  (explicit TLS => STARTTLS, Authentication is DISABLED => use port 465/587 instead)
      - "143:143"  # IMAP4 (explicit TLS => STARTTLS)
      - "465:465"  # ESMTP (implicit TLS)
      - "587:587"  # ESMTP (explicit TLS => STARTTLS)
      - "993:993"  # IMAP4 (implicit TLS)
    volumes:
      - /home/lyxas/docker_data/dms/mail-data/:/var/mail/
      - /home/lyxas/docker_data/dms/mail-state/:/var/mail-state/
      - /home/lyxas/docker_data/dms/mail-logs/:/var/log/mail/
      - /home/lyxas/docker_data/dms/config/:/tmp/docker-mailserver/
      - /home/lyxas/certs:/tmp/dms/custom-certs:ro
      - /etc/localtime:/etc/localtime:ro
    restart: always
    stop_grace_period: 1m
    # Uncomment if using `ENABLE_FAIL2BAN=1`:
    # cap_add:
    #   - NET_ADMIN
    healthcheck:
      test: "ss --listening --tcp | grep -P 'LISTEN.+:smtp' || exit 1"
      timeout: 3s
      retries: 0`

Thanks in advance for your help
Lyxas

**Comment by casperklein** [comment, marked as answer]
Try `SSL_TYPE=manual` instead.

**Reply by Lyxass** [reply]
Hey casperklein,

Thank you for your answer, indeed I confused manual with custom. it works now thanks

Lyxas
